### PR TITLE
Applab intro show notes

### PR DESCRIPTION
**Pull request by caleybrock**
doc from GT: https://docs.google.com/document/d/1BU3OhRoLQSFXC_C3i_ClwFXdukj6JTW1HaivoilIYDw/edit

### CODE REVIEW

File: dashboard/config/locales/slides.en.yml [hunk 1]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+        text: "From here you can either upload a sound file from your computer, or search for a sound from the sound library. The sound library has lots of different categories like instruments, background music or animals."
+      4:
+        image: 'notes/applab_intro_sounds_4.png'
+        text: "Once you've got the sound you want, click \"Choose\". When this block runs, it will play the sound you chose."
+      5:
+        image: 'notes/applab_intro_sounds_5.png'
+        text: "To add images to your elements you can just use the setProperty block. Select the image property in the second drop-down"
+      6:
+        image: 'notes/applab_intro_sounds_6.png'
+        text: "Then select "Choose" from the third drop down."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: Need to escape `\"Choose\"`

File: dashboard/config/locales/slides.en.yml [hunk 2]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+        text: "You can also share a link to your app over social media or post it to the code.org public gallery for other people to play and share."
+      6:
+        image: 'notes/applab_intro_share_6.png'
+        text: "You've just scratched the surface of what App Lab can do. Full App Lab features more advanced user interactions, new screen elements and even a built-in database so you can quickly build full featured apps. At \"code.org/applab\" you'll find the full tool and lots of ideas if you want to keep going."
+      7:
+        image: 'notes/applab_intro_share_7.png'
+        text: "Thanks for coming along the ride. Now go build and share your first app, and keep on building apps with App Lab!"
+
+
+
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml:1916]: I don't know why GitHub is saying there is no new line at the end of the file but I see several spaces. Not sure if fixable.

File: dashboard/config/locales/slides.en.yml [hunk 3]
```diff
@@ -1768,3 +1768,149 @@ en:
       2:
         image: 'notes/mc_2016_congrats_2.jpg'
         text: "When you're done making your game, choose share to get a link that you can share with your friends or to play your game on your phone. Have fun! I'm looking forward to playing your version of Minecraft."
+    applab_intro_welcome:
+      1:
+        image: 'notes/applab_intro_welcome_1.png'
+        text: "My name is Mei'lani. I'm an intern at code.org and I'm also a computer science student. I like computer science because it's really creative. I get to be the problem solver. I get to make everything function how I want it to. But I also get to be the designer. I get to be the artist and choose everything on the creative side of it."
+      2:
+        image: 'notes/applab_intro_welcome_2.png'
+        text: "Making your own apps is easy with App Lab! Whether you're new to coding or have some experience. App Lab Apple is a great tool for building new apps and sharing them with your friends."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: App Lab Apple?

Comment (tanyaparker) [reply dashboard/config/locales/slides.en.yml]: If it's wrong here, may need to also be updated in the caption track.

File: dashboard/config/locales/slides.en.yml [hunk 4]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+        text: "Once you're done, the block reads a bit like a sentence:'Set button1's background color to green.' Hit run to see the changes you're building in code. If you reset you can drag in more blocks to change other things about your app. Like the text!"
+      8:
+        image: 'notes/applab_intro_welcome_8.png'
+        text: "For this tutorial we'll be working in block mode but App Lab also supports working in text."
+      9:
+        image: 'notes/applab_intro_welcome_9.png'
+        text: "Either way you'll be programming in JavaScript, the language of the web. Each level has a goal, instructions and an image of what you're aiming to create."
+      10:
+        image: 'notes/applab_intro_welcome_10.png'
+        text: "If you're ever stuck click on this picture and it will show you exactly how to complete the level. But see if you can do it on your own first."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: Comma after stuck

Comment (tanyaparker) [reply dashboard/config/locales/slides.en.yml]: Not clear what picture to click on. This isn't blocking merging but would recommend drawing a red rectangle with an arrow around what she's talking about.

File: dashboard/config/locales/slides.en.yml [hunk 5]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+      11:
+        image: 'notes/applab_intro_welcome_11.png'
+        text: "Once you've completed the goal for this level click finish to move on. That's all there is to it! Hit the orange \"continue\" button to go to the next level and start your adventure with an App Lab!"
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: I usually capitalize `Finish` like the button.

File: dashboard/config/locales/slides.en.yml [hunk 6]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+    applab_intro_events:
+      1:
+        image: 'notes/applab_intro_events_1.png'
+        text: "Apps are really cool because it's kind of like starting your own business I mean, you get to design everything from the ground up, you get to manage everything, and you're the owner of this piece of work that you just created with code."
+      2:
+        image: 'notes/applab_intro_events_2.png'
+        text: "Now that you can change the look of your app let's make it interactive by responding to events! Events are user actions like clicking a button, scrolling through a menu, or hovering over a picture. Interactive apps need ways to respond to events, like playing a sound when I click this button. To do this in App Lab you need to use a new block called onEvent."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: Not a ship blocker, but we usually add quotes around all block names `"onEvent"`

File: dashboard/config/locales/slides.en.yml [hunk 7]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+        text: "To add images to your elements you can just use the setProperty block. Select the image property in the second drop-down"
+      6:
+        image: 'notes/applab_intro_sounds_6.png'
+        text: "Then select "Choose" from the third drop down."
+      7:
+        image: 'notes/applab_intro_sounds_7.png'
+        text: "From here you can upload an image from your computer or you can look through a huge library of icons in the icon library."
+      8:
+        image: 'notes/applab_intro_sounds_8.png'
+        text: "Back in code mode you can use the setProperty block to change the icon color of your icon. Once you've picked what image your icon to use click \"Run\" to see how it looks."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: `"setProperty"`?

File: dashboard/config/locales/slides.en.yml [hunk 8]
```diff
@@ -1768,3 +1768,149 @@ en:
...
+      9:
+        image: 'notes/applab_intro_sounds_9.png'
+        text: "That's all there is to it! Now you can start adding images and sounds to make your apps even more fun and dynamic."
+    applab_intro_designmode:
+      1:
+        image: 'notes/applab_intro_designmode_1.png'
+        text: "To build your own apps you're going to need to start designing screens and elements from scratch. App Lab makes this easy to do with Design Mode."
+      2:
+        image: 'notes/applab_intro_designmode_2.png'
+        text: "Use the switch on top of your app to go into design mode."
```
Comment (tanyaparker) [inline dashboard/config/locales/slides.en.yml]: We capitalized Design Mode in the previous text but not this one?

**Comment by breville** [comment]
I'm excited to see my plant in the background

**Pull request merged**